packages/block-editor/src/components/iframe/index.js

```diff
@@ -251,13 +260,28 @@ function Iframe( {
 			>
 				{ iframeDocument &&
 					createPortal(
+						// We want to prevent React events from bubbling throught the iframe
+						// we bubble these manually.
+						/* eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions */
 						<body
 							ref={ bodyRef }
 							className={ classnames(
 								'block-editor-iframe__body',
 								'editor-styles-wrapper',
 								...bodyClasses
 							) }
+							onKeyDown={ ( event ) => {
+								// This stopPropagation call ensures React doesn't create a syncthetic event to bubble this event
+								// which would result in two React events being bubbled throught the iframe.
+								event.stopPropagation();
+								const { defaultView } = iframeDocument;
+								const { frameElement } = defaultView;
+								bubbleEvent(
+									event,
+									window.KeyboardEvent,
+									frameElement
+								);
+							} }
```

Comment:
Why are we listening for the React event instead of native event?

Reply:
That's the only way to basically prevent the "double event bubbling" properly. If I use `stopPropagation` on a regular event handler and I trigger the event on the frame, it never reached the upper event handlers in my tests.

Reply:
There's something weird about this, but I guess we can investigate later.